data/changelog.txt

```diff
@@ -108,6 +122,11 @@ Bombastic Perks adds Playstyle Perks
 Disable the Bionic Professions mod by default
 Add the Mind Over Matter mod to the CDDA repository
 [Tamable Wildlife] More tamable creatures
+Add Zener cards
+Tamable Wildlife: tamable rattlesnakes
+Resubmission of #47984 with fixes and a roof
+[DinoMod] giganotosaurus
+Add noetic resilience treatment, fix https://github.com/CleverRaven/Cataclysm-DDA/issues/66804
```

Comment:
```suggestion
```
Not sure about these. Some are additions to content that was added post-0.G and others are small or not well described.